frontend/src/views/category/Category.vue

```diff
@@ -0,0 +1,156 @@
+            <h3>
+              {{ $tc('views.category.category.properties') }}
+            </h3>
+          </v-expansion-panel-header>
+          <v-expansion-panel-content>
+            <CategoryProperties :category="category()" :disabled="!isManager" />
+          </v-expansion-panel-content>
+        </v-expansion-panel>
+
+        <v-expansion-panel>
+          <v-expansion-panel-header>
+            <h3>
+              {{ $tc('views.category.category.template') }}
+            </h3>
+          </v-expansion-panel-header>
+          <v-expansion-panel-content>
+            <CategoryTemplate
+              :category="category()"
+              :layout-mode="layoutMode"
+              :loading="loading"
+              :readonly="!isManager"
+            />
+          </v-expansion-panel-content>
+        </v-expansion-panel>
+      </v-expansion-panels>
+    </content-card>
+  </v-container>
+</template>
+
+<script>
+import ContentCard from '@/components/layout/ContentCard.vue'
+import CategoryChip from '@/components/generic/CategoryChip.vue'
+import DialogEntityDelete from '@/components/dialog/DialogEntityDelete.vue'
+import { campRoleMixin } from '@/mixins/campRoleMixin.js'
+import ErrorExistingActivitiesList from '@/components/campAdmin/ErrorExistingActivitiesList.vue'
+import CategoryProperties from '@/components/category/CategoryProperties.vue'
+import CategoryTemplate from '@/components/category/CategoryTemplate.vue'
+export default {
+  name: 'Category',
+  components: {
+    CategoryTemplate,
+    CategoryProperties,
+    ErrorExistingActivitiesList,
+    DialogEntityDelete,
+    CategoryChip,
+    ContentCard,
+  },
+  mixins: [campRoleMixin],
+  provide() {
+    return {
+      preferredContentTypes: () => this.preferredContentTypes,
+      allContentNodes: () => this.contentNodes,
+      camp: () => this.camp(),
+    }
+  },
+  props: {
+    camp: {
+      type: Function,
+      required: true,
+    },
+    category: {
+      type: Function,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      layoutMode: true,
+      loading: true,
+      openPanels: [0, 1],
+    }
+  },
+  computed: {
+    contentNodes() {
+      return this.category().contentNodes()
+    },
+    preferredContentTypes() {
+      return this.category().preferredContentTypes()
+    },
+  },
+
+  // reload data every time user navigates to Category view
+  async mounted() {
+    // hide properties panel if new category has been created and user is redirected here
+    const url = new URL(window.location)
+    if (url.searchParams.get('new') === 'true') {
+      url.searchParams.delete('new')
+      window.history.replaceState({}, '', url)
+      this.openPanels = [1]
+    }
+    this.loading = true
+    await this.category()._meta.load // wait if category is being loaded as part of a collection
+    await this.category().$reload() // reload as single entity to ensure all embedded entities are included in a single network request
```

Comment:
if this fails, the loading state never resolves.
Maybe use a computed of this.category()._meta.loading?
Maybe the error is already catched in the global error handler and shown, else i would catch the error here manually and show a toast

Reply:
Correct. I think we should look into this in a followup issue, as this behavior was already present before. Can you open an issue for this?